microsoft/react-native-test-app
# feat: add ability to set app version

### Description

Adds ability to set app version.

Partially resolves #990.

### Platforms affected


### Test plan

1. Set memorable version numbers such as:
   ```diff
   diff --git a/example/app.json b/example/app.json
   index ef69fa7..1173f1b 100644
   --- a/example/app.json
   +++ b/example/app.json
   @@ -2,6 +2,7 @@
      "$schema": "https://raw.githubusercontent.com/microsoft/react-native-test-app/trunk/schema.json",
      "name": "Example",
      "displayName": "Example",
   +  "version": "2.0",
      "components": [
        {
          "appKey": "Example",
   @@ -13,6 +14,15 @@
          "presentationStyle": "modal"
        }
      ],
   +  "android": {
   +    "versionCode": "9000"
   +  },
   +  "ios": {
   +    "buildNumber": "9000"
   +  },
   +  "macos": {
   +    "buildNumber": "9000"
   +  },
   ```
2. Build Android app and verify that the version numbers are correctly set: `yarn android`
3. Prepare iOS/macOS Xcode projects and verify that the version numbers are correctly set: `pod install`
   ![Screenshot 2022-07-14 at 16 03 26](https://user-images.githubusercontent.com/4123478/179005505-be94f66c-9cb3-4aca-92e3-4a73ff6c25d6.png)
